azure/template.json

```diff
@@ -0,0 +1,199 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
serviceName parameter is missing -> https://github.com/SkillsFundingAgency/das-platform-building-blocks#resource-naming